Pylint: Fix the `use-maxsplit-arg` issues.

## 💡 Description
Update the codebase to use the `maxsplit` argument in `split` calls when looking only for a particular part of the string. `pylint` report is attached (`pylint . --disable=all --enable=use-maxsplit-arg`):
```
************* Module pds.naif_pds4_bundler.classes.bundle
pds/naif_pds4_bundler/classes/bundle.py:406:33: C0207: Use self._vid.split('.', maxsplit=1)[0] instead (use-maxsplit-arg)
pds/naif_pds4_bundler/classes/bundle.py:508:26: C0207: Use product.rsplit('.', maxsplit=1)[-1] instead (use-maxsplit-arg)
pds/naif_pds4_bundler/classes/bundle.py:644:26: C0207: Use product.rsplit('.', maxsplit=1)[-1] instead (use-maxsplit-arg)
************* Module pds.naif_pds4_bundler.classes.plan
pds/naif_pds4_bundler/classes/plan.py:221:30: C0207: Use line.rsplit(' - ', maxsplit=1)[-1] instead (use-maxsplit-arg)
```

--- a/src/pds/naif_pds4_bundler/classes/bundle.py
+++ b/src/pds/naif_pds4_bundler/classes/bundle.py
@@ -403,7 +403,7 @@ class Bundle:
         # The number of previous releases is obtained from the number/version
         # of Bundle labels. That information is already known as it is
         # specified by the bundle vid.
-        number_of_releases = int(self._vid.split(".")[0])
+        number_of_releases = int(self._vid.split(".", maxsplit=1)[0])
 
         # When the pipeline has not yet run, the in-progress release does not
         # yet have a label on disk, so we only reconstruct completed releases.
@@ -505,7 +505,7 @@ class Bundle:
                     product = (
                         f"spice_kernels/{line.split(':')[5].replace('_', '/', 1)}"
                     )
-                    ext = product.split(".")[-1]
+                    ext = product.split(".", maxsplit=1)[-1]
                     products.append(product)
                     products.append(product.replace(f".{ext}", ".xml"))
 
@@ -641,7 +641,7 @@ class Bundle:
                 if ":checksum_" not in line:
                     # OrbNums: preserve original extension
                     product = f"miscellaneous/{product_name}"
-                    ext = product.split(".")[-1]
+                    ext = product.split(".", maxsplit=1)[-1]
                     products.append(product)
                     products.append(product.replace(f".{ext}", ".xml"))
 

--- a/src/pds/naif_pds4_bundler/classes/plan.py
+++ b/src/pds/naif_pds4_bundler/classes/plan.py
@@ -218,7 +218,7 @@ class ReleasePlan:
             logging.warning(f"-- {line}.")
             logging.info("")
             if not self.setup.args.silent and not self.setup.args.verbose:
-                print("-- " + line.split(" - ")[-1] + ".")
+                print("-- " + line.split(" - ", maxsplit=1)[-1] + ".")
 
             return False
 

